service/lib/ops/policyOps.js

```diff
@@ -39,16 +39,26 @@ function insertPolicies (client, policies, cb) {
   client.query(sql, cb)
 }
 
-function deletePolicies (client, ids, cb) {
-  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids})`, utils.boomErrorWrapper(cb))
+function deletePolicies (client, ids, orgId, cb) {
+  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids}) AND org_id=${orgId}`, utils.boomErrorWrapper(cb))
```

Comment:
Sorry, I wasn't clear.

If the ids are not all from the same organization, we should return either a 400 or a 404. I was wondering if we are doing it somewhere else that is not in this file. If we don't, I think we should.